miio/tests/test_yeelight.py

```diff
@@ -66,11 +81,11 @@ def test_status(self):
         assert repr(status) == repr(YeelightStatus(self.device.start_state))
 
         assert status.name == self.device.start_state["name"]
-        assert status.is_on is False
-        assert status.brightness == 100
-        assert status.color_temp == 3584
-        assert status.color_mode == YeelightMode.ColorTemperature
-        assert status.rgb is None
+        assert status.lights[0].is_on is False
+        assert status.lights[0].brightness == 100
+        assert status.lights[0].color_temp == 3584
+        assert status.lights[0].color_mode == YeelightMode.ColorTemperature
+        assert status.lights[0].rgb is None
         assert status.developer_mode is True
```

Comment:
It would be great if the original property names would keep working, just internally defaulting to the main light.

Reply:
Look 5255611